app/javascript/controllers/clear_preview_controller.js

```diff
@@ -0,0 +1,11 @@
+import { Controller } from '@hotwired/stimulus';
+
+export default class ClearPreviewController extends Controller {
+  static targets = ['input', 'write'];
+
+  clear() {
```

Comment:
Nit: Instead of introducing a new controller to do this, we could make the button do a hard refresh to simulate the same behaviour.

```ruby
<%= button_to lessons_preview_path, class: 'button button--secondary py-2 px-3', method: :get do %>
...
<% end %>
```

Unless theres a good reason to do this programmatically?

Reply:
Don't think so, no. I could see an argument be made for the entire tool to be usable on spotty/non-existent Internet connection but that'd basically mean changing the way the entire page works.